.github/workflows/test.yaml

```diff
@@ -102,19 +103,45 @@ jobs:
           # To ignore a bad cache, bump the cache* integer.
           gha-cache-key: cache0-py${{ matrix.python-version }}
 
-      - name: Test
+      - name: Test pants-plugins
+        if: ${{ matrix.python-version-short == '3.9' }}
         env:
           # Github Actions uses the 'runner' user, so use that instead of stanley.
           ST2TESTS_SYSTEM_USER: 'runner'
-        # We do not support running pytest everywhere yet. When we do it will be simply:
-        #   pants test ::
-        # Until then, we need to manually adjust this command line to test what we can.
         run: |
-          pants test pylint_plugins/:: pants-plugins/::
+          pants test pants-plugins/::
+
+      # We do not support running pytest everywhere yet. When we do it will be simply:
+      #   pants test ::
+      # Until then, we need to manually adjust this command line to test what we can.
+      # So far, this includes unit, pack, and pants-plugins tests.
+      # TODO: run integration tests
+
+      - name: Unit Tests
+        env:
+          # Github Actions uses the 'runner' user, so use that instead of stanley.
+          ST2TESTS_SYSTEM_USER: 'runner'
+        run: >
+          pants
+          --python-bootstrap-search-path=[]
+          --python-bootstrap-search-path=${{ steps.python.outputs.python-path }}
+          --tag=unit
+          test '::'
```

Comment:
`::` is a glob that means everything. Using `--tag=unit` filters everything down to only those test targets that are tagged with `unit`.